man/kernel-command-line.xml

```diff
@@ -585,6 +585,13 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>hostname=</varname></term>
+
+        <listitem><para>Accepts a hostname to set during early boot. It is used as a default until another
+        hostname is set via e.g. <filename>/etc/hostname</filename> or DHCP.</para></listitem>
```

Comment:
should have a comment somewhere that this is implemented by both the kernel itself and systemd, too

Reply:
I've added the following sentence:

https://github.com/systemd/systemd/blob/4c1cfc72aef1f6ee6a7d0529ba54a48f8a102b93/man/kernel-command-line.xml#L594-L595

Does that sound good to you?